src/main/java/com/coveo/saml/SamlClient.java

```diff
@@ -539,41 +438,27 @@ public synchronized static void ensureOpenSamlIsInitialized() throws SamlExcepti
     }
   }
 
-  private static DOMParser createDOMParser() throws SamlException {
-    DOMParser parser =
-        new DOMParser() {
-          {
-            try {
-              setFeature(INCLUDE_COMMENTS_FEATURE, false);
-            } catch (Throwable ex) {
-              throw new SamlException(
-                  "Cannot disable comments parsing to mitigate https://www.kb.cert.org/vuls/id/475445",
-                  ex);
-            }
-
-            try {
-              setFeature(XERCES_FEATURE_PREFIX + DISALLOW_DOCTYPE_DECL_FEATURE, true);
```

Comment:
I see that you did some research to ensure that we wouldn't "lose" this fix - it looks alright to me, especially since the UTs continue to pass unchanged.